rules/retrieve.smk

```diff
@@ -732,9 +732,9 @@ if config["enable"]["retrieve"]:
         params:
             default_cutout=config_provider("atlite", "default_cutout"),
         output:
-            seawater_temperature="data/seawater_temperature_{year}.nc",
+            seawater_temperature="data/seawater_temperature_{planning_horizons}.nc",
```

Comment:
```suggestion
            seawater_temperature="data/seawater_temperature_{year}.nc",
```
No, using `planning_horizons` here would be confusing because the wildcard refers to a weather year.